condensation-forum/application.py

```diff
@@ -19,7 +19,10 @@
 from data import query, schema
 from forms import CreateThreadForm, CreateCommentForm
 import inspect
-
+from werkzeug.utils import secure_filename
+import hashlib
```

Comment:
we no longer use hashlib, remove any unused impoorts

Reply:
Fixed